index.js

```diff
@@ -278,8 +278,17 @@
          * Store Injection
          */
         function createStoreInjector(stores, component) {
+            
+            var statics = Object.getOwnPropertyNames(component).filter(function (propName) {
+				return typeof component[propName] === 'function';
```

Comment:
@mattruby Just wanted to give a heads up, this fails in Safari because `Object.getOwnPropertyNames` apparently doesn't respect strict mode and returns `arguments` and `caller` also as properties. It would perhaps have been better to use `Object.keys` here.